src/app/createlog/page.tsx

```diff
@@ -0,0 +1,232 @@
+"use client";
+import { useAuthSession } from "@/lib/contexts/auth-context/auth-context";
+import { ExercisesDictionary } from "@/lib/exercises/exercises-dictionary";
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import moduleStyles from "../home-page.module.css";
+
+export default function CreateLog() {
```

Comment:
Typescript is complaining a lot in this file, likely because there are no types being annotated. LMK if you need help with this?